libs/controller/controller.py

```diff
@@ -831,7 +851,24 @@ def is_fw_available(self, fw_version=None):
                 firmware_version_name=fw_version + ".tar.gz")
             firmware_version = firmware_version._id
             print("Firmware ID: ", firmware_version)
-        except:
+        except Exception as e:
+            print(e)
             firmware_version = False
             print("firmware not available: ", firmware_version)
         return firmware_version
+
+
+if __name__ == '__main__':
+    """
+    Examples to Try Out
+    """
+    controller = {
+        'url': "https://wlan-portal-svc-nola-ext-03.cicd.lab.wlan.tip.build",  # API base url for the controller
```

Comment:
These cluster may not always be there. Suggest to move them to input.

Reply:
these are just for reference as a format